torchaudio/utils/ffmpeg_utils.py

```diff
@@ -0,0 +1,47 @@
+import torch
+
+
+def get_log_level() -> int:
+    """Get the log level of FFmpeg.
+
+    See :py:func:`set_log_level` for the detailo.
+    """
+    return torch.ops.torchaudio.ffmpeg_get_log_level()
+
+
+def set_log_level(level: int):
+    """Set the log level of FFmpeg (libavformat etc)
+
+    Arguments:
+        level (int): Log level. The larger, the more verbose.
+
+            The following values are valid values, the corresponding ``ffmpeg``'s
```

Comment:
what happens if they enter an invalid value? should we add a check?

Reply:
I think they will boil down to a simple level comparison, so it's safe to put other values.

https://ffmpeg.org/doxygen/4.1/log_8c_source.html#l00316

It's just normally, log-levels are set by `ffmpeg` CLI, which exposes the listed values. Let me rephrase, `common values`, instead of saying `valid values`.